ads/3p.md

```diff
@@ -0,0 +1,99 @@
+### Instructions to integrate, if you are:
```

Comment:
Replace this header with a quick overview.  e.g.:

If you are an ad technology provider looking to integrate with AMP HTML, please see the following guidelines: